app/scripts/proactive/view/dom.js

```diff
@@ -21,6 +21,7 @@ define(
         'codemirror/mode/powershell/powershell',
         'codemirror/mode/r/r',
         'codemirror/mode/yaml/yaml',
+        //'codemirror/addon/simplemode/dockerfile', 
```

Comment:
if it does not work, try to put instead:
'codemirror/addon/simple',
'codmirror/mode/dockerfile'